packages/core/src/Stores/client-store.js

```diff
@@ -1644,11 +1645,13 @@ export default class ClientStore extends BaseStore {
                 BinarySocketGeneral.authorizeAccount(authorize_response);
 
                 // Client comes back from oauth and logs in
-                RudderStack.identifyEvent(user_id, {
-                    language: getLanguage().toLowerCase(),
+                Analytics.setAttributes({
+                    user_language: getLanguage().toLowerCase(),
+                    app_id: getAppId(),
+                    user_id,
                 });
                 const current_page = window.location.hostname + window.location.pathname;
-                RudderStack.pageView(current_page);
+                Analytics.pageView(current_page);
```

Comment:
sorry that return to this again, but pageView have no sense here because it's core package without any interface
isn't it?

Reply:
`pageView` has been added recently here : https://github.com/binary-com/deriv-analytics/blob/master/src/analytics.ts

Reply:
I mean client-store.js have no ui, why we track that page is viewed?